semperai/arbius
# Development branch of Weboasis-arbius to be merged for the new website.

## Summary
### This pull request encompasses the comprehensive revamp of the Arbius website, utilizing Next.js to enhance functionality and user experience.
### The update aims to create a fully mobile-friendly interface that adapts seamlessly across different devices.
## Testing Instructions To test the application locally:
1. Navigate to the website directory.
2. Run the application. These steps will allow you to review the updated features and ensure the website operates smoothly on various devices.
